services/worker/src/worker/job_runners/config/parquet_and_info.py

```diff
@@ -128,20 +128,17 @@ def __init__(
 
     @property
     def path_in_repo(self) -> str:
-        if self.partial:
-            # Using 4 digits is ok since MAX_FILES_PER_DIRECTORY == 10_000
-            return f"{self.config}/partial/{self.split}/{self.shard_idx:04d}.parquet"
-        else:
-            return f'{self.config}/{self.local_file.removeprefix(f"{self.local_dir}/")}'
+        # Using 4 digits is ok since MAX_FILES_PER_DIRECTORY == 10_000
+        partial_path = "/partial" if self.partial else ""
+        return f"{self.config}{partial_path}/{self.split}/{self.shard_idx:04d}.parquet"
```

Comment:
Maybe a dummy question but I was thinking that a "partial" was like a preview of a split, not sure if it should go inside a config.

Reply:
Same: I think it would be easier to understand as
```suggestion
        return f"{self.config}/{self.split}{partial_path}/{self.shard_idx:04d}.parquet"
```

Reply:
+ we avoid conflict if a split is name `partial`, and we avoid the user to have to think if it's a split name or the special partial keyword

Reply:
makes sense, I'll do the change